package.json

```diff
@@ -69,6 +69,7 @@
     "jest-environment-jsdom": "^29.5.0",
     "jsdom": "^22.0.0",
     "prettier": "^2.8.7",
+    "react": "^16.14.0",
```

Comment:
Do we need this dependency if that doesn't need to be a TSX file?

Reply:
`Components.tsx` actually use `React` with the `useUpdateWithSetter` [1]. By only including it in the `peerDependencies` [2] it won't get picked up by `yarn`. A person would need to install all dependencies with:

```
yarn install --install-peers
```

I think it makes sense to include it in the `devDependencies` because you need `React` to be installed to actually develop/test the project.

[1] https://github.com/stripe/react-connect-js/blob/master/src/utils/useUpdateWithSetter.ts
[2] https://github.com/stripe/react-connect-js/blob/master/package.json#L88